src/vm/docs/mod.rs

```diff
@@ -737,6 +747,114 @@ definition (i.e., you cannot put a define statement in the middle of a function
 "
 };
 
+const MINT_TOKEN: SpecialAPI = SpecialAPI {
+    name: "mint-token!",
+    input_type: "TokenName, int, principal",
+    output_type: "(response bool int)",
+    signature: "(mint-token! token-name amount recipient)",
+    description: "`mint-token!` is used to increase the token balance for the `recipient` principal for a token
+type defined using `define-token`. The increased token balance is _not_ transfered from another principal, but
+rather minted.",
```

Comment:
Why is the output type `(response bool int)`?  What is the `bool` and what is the `int`?